src/terms/hartree.jl

```diff
@@ -26,17 +26,26 @@ struct TermHartree <: TermNonlinear
     # Fourier coefficients of the Green's function of the periodic Poisson equation
     poisson_green_coeffs::AbstractArray
 end
-function TermHartree(basis::PlaneWaveBasis{T}, scaling_factor) where {T}
+function coeffs_poisson_hartree(basis::PlaneWaveBasis{T}, scaling_factor;
```

Comment:
`compute_poisson_green_coeffs` to make the connection to `poisson_green_coeffs` clear ?